### DISCUSSION TITLE
v1.1.0

### DISCUSSION THREAD

**Discussion by dotargz** [dotargz/novel-engine#6] [Announcements]
## What's Changed
* Added simple "anticheat" to protect engine files
* Added documentation on the wiki
* Lots of small bug fixes and improvements
* Really small README.md change that will improve it tenfold by @devxan in https://github.com/dotargz/novel-engine/pull/5

## New Contributors
* @devxan made their first contribution in https://github.com/dotargz/novel-engine/pull/5

**Full Changelog**: https://github.com/dotargz/novel-engine/commits/v1.1.0

<hr /><em>This discussion was created from the release <a href='https://github.com/dotargz/novel-engine/releases/tag/v1.1.0'>v1.1.0</a>.</em>

**Comment by devxan** [comment]
Why does GitHub create discussions based on releases 🗣️
